apps/csm-portal/backend/cmd/server/main.go

```
@@ -220,19 +240,25 @@ func main() {
 	mux.HandleFunc("POST /problems/search", problemHandler.SearchProblems)
 
+	// Built once and reused on both listeners below: Auth() does a real JWKS
+	// fetch (when TokenValidatorEnabled), so calling it a second time would
+	// duplicate that startup network round-trip and double the chance of a
+	// transient JWKS hiccup aborting startup, for no benefit — both
+	// listeners validate the exact same tokens the exact same way.
+	authMiddleware := middleware.Auth(authCfg)
+
 	addr := ":" + mustPort("PORT", "8080")
 
 	ln, err := net.Listen("tcp", addr)
```

golangci-lint (2.12.2)

[error] net.Listen must not be called. use (*net.ListenConfig).Listen

(noctx)